src/components/ClockName.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+interface Props {
+  name: number;
+}
+
+export class ClockName extends React.Component<Props> {
+  state = {
+    clockName: this.props.name,
```

Comment:
You should pass clockName as props in Clock component
and using this.props.clockName get needed value and render on the page